lib/LaTeXML/Post.pm

```diff
@@ -447,7 +447,7 @@ sub convertXMTextContent {
   foreach my $node (@nodes) {
     if ($node->nodeType == XML_TEXT_NODE) {
       my $string = $node->textContent;
-      $string =~ s/^\s+/$NBSP/; $string =~ s/\s+$/$NBSP/;    # should we???
```

Comment:
This line of code seems to add non-breaking spaces which are presented in omdoc in a strange way. 

``` 
&#xA0;and&#xA0;
```
The line-breaks in this context should't matter.

That's why I commented this out.